docs/readthedocs/source/doc/Nano/Overview/index.rst

```diff
@@ -3,6 +3,7 @@ Nano Key Features
 
 * `PyTorch Training <pytorch_train.html>`_
 * `PyTorch Inference <pytorch_inference.html>`_
+* `PyTorch CUDA patch <cuda_patch.html>`_
```

Comment:
Seems like you forgot to change the file name here :)
```rst
* `PyTorch CUDA Patch <pytorch_cuda_patch.html>`_
```

Reply:
changed it, thanks